rich/progress.py

```diff
@@ -1585,16 +1647,16 @@ def remove_task(self, task_id: TaskID) -> None:
         *Progress.get_default_columns(),
         TimeElapsedColumn(),
         console=console,
-        transient=True,
+        transient=False,
```

Comment:
I'm assuming the code below was for testing. Can you restore this so that `progress -m rich.progress` show a demonstration of multiple progress bars.